data-table/__tests__/table.js

```diff
@@ -1,9 +1,65 @@
+// test 4-5 function
 const table = require('../table.js');
 
 beforeEach(() => {
     document.body.innerHTML = '<div id="div-id"></div>'
 });
 
+describe('test createDataTable', () => {
```

Comment:
Needs a better description - seems to be that everything will test createDataTable. Perhaps "basic tests to ensure ______"